localstack/config.py

```diff
@@ -244,6 +244,10 @@ def is_env_not_false(env_var_name):
 # whether the in_docker check should always return true
 OVERRIDE_IN_DOCKER = is_env_true("OVERRIDE_IN_DOCKER")
 
+# number of times proxy will retry the request to the backend service. Set to 0 to disable retries (especially useful for debugging)
+_proxy_retries = os.environ.get("PROXY_MAX_RETRIES")
+PROXY_MAX_RETRIES = None if _proxy_retries is None else int(_proxy_retries)
```

Comment:
nit: Let's better hedge for non-`None` values like empty strings (we may see values like `PROXY_MAX_RETRIES=` getting passed in, causing the `int` cast to fail):
```
_proxy_retries = str(os.environ.get("PROXY_MAX_RETRIES") or "").strip()
PROXY_MAX_RETRIES = None if not _proxy_retries else int(_proxy_retries)
```